styles/output.css.map

```diff
@@ -1 +1 @@
-{"version":3,"sourceRoot":"","sources":["input.scss","artist-aside.scss","artist-images.scss","artists-page.scss"],"names":[],"mappings":"AAGQ;ACHR;EACE;EACA;EACA;EACA;EACA;EACA;EACA;;;AAGF;EACE;EACA;EACA;EACA;;;AAGF;EACE;;;AAGF;EACE;EACA;;;AAGF;EACE;EACA;EACA;EACA;;;AAGF;EACE;IACE;IACA;;EAGF;IACE;IACA;IACA;IACA;IACA;IACA;IACA;IACA;;EAOF;IACE;;EAGF;IACE;;;AAIJ;EACE;IACE;IACA;;EAGF;IACE;IACA;;EAGF;IACE;IACA;IACA;IACA;IACA;;;AC/EJ;EACE;EACA;EACA;;;AAGF;EACE;;;AAGF;EACE;EACA;;;AAGF;EACE;;;AAGF;EACE;;;AAGF;EACE;IACE;IACA;IACA;;EAEF;IACE;;;AAIJ;EACE;IACE;;;AAIJ;EACE;IACE;IACA;IACA;IACA;IACA;IACA;IACA;IACA;;EAGF;IACE;;;ACrDJ;EACE;EACA;EACA;EACA;;;AAGF;EACE;;;AHHF;EACE;;;AAGF;EACE;;;AAGF;EACE;EACA;;;AAGF;EACE;EACA;EACA;EACA","file":"output.css"}
```

Comment:
This causes a merge conflict, since the main  branch say it should be deleted. We can look at how to resolve merge conflicts together.